apps/hpc-ftsadmin/src/app/utils/parse-filters.ts

```diff
@@ -55,24 +58,30 @@ export type FlowStatusType =
   | 'pass_through'
   | 'standard';
 
-/**
+/*
  * Type guard functions
  */
 
-const filterValueIsString = (value: FilterValues | number): value is string => {
+const isEmptyValue = (value: unknown): value is EmptyValue =>
+  value === null ||
+  value === false ||
+  (typeof value === 'string' && value === '') ||
+  (Array.isArray(value) && value.length === 0);
+
+const filterValueIsString = (value: FilterValue | number): value is string => {
   return typeof value === 'string';
 };
 
-const filterValueIsArrayString = (value: FilterValues): value is string[] => {
+const filterValueIsArrayString = (value: FilterValue): value is string[] => {
   return Array.isArray(value) && typeof value[0] === 'string';
 };
```

Comment:
I'd be more strict and check that every element is of type string. Also, naming it "string array" may be better
```suggestion
const filterValueIsStringArray = (value: FilterValue): value is string[] => {
  return Array.isArray(value) && value.every((item) => typeof item === 'string');
};
```